client/src/Overview/Components/HorizontalCarousel.jsx

```diff
@@ -2,7 +2,6 @@ import React, { useState } from 'react';
 import { HorizontalImage, HorizontalSlider } from './HorizontalScrollEntry.js';
 import RightButton from './RightButton.jsx';
 import LeftButton from './LeftButton.jsx';
-//import ExpandShrinkButton from './ExpandShrinkButton.jsx';
 
 
 const HorizontalCarousel = () => {
```

Comment:
I just wanted to confirm for the carousel that both left and right buttons are rendered at all times rather than conditionally if they are at last image or first image.